infrastructures/infrastructure-common/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/util/LinuxInitScriptGenerator.java

```diff
@@ -36,6 +36,8 @@
 
 public class LinuxInitScriptGenerator {
 
+    public static final String DEFAULT_SUFFIX_RM_TO_NODEJAR_URL = ":8080/rest/node.jar";
```

Comment:
I suggest to remove this default value (DEFAULT_SUFFIX_RM_TO_NODEJAR_URL = ":8080/rest/node.jar").

We don't needed it, if the configuration is not properly loaded, we throw an exception.

Reply:
Ok